### PR TITLE
Feature/zk UI 141 handle errors and loading states

### PR DESCRIPTION
**Pull request by JBWatenbergScality**

### CODE REVIEW

File: src/react/account/__tests__/Accounts.test.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import {
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
```
Comment (ChengYanJin) [inline src/react/account/__tests__/Accounts.test.tsx]: can be removed ;)

**Review by ChengYanJin — APPROVED** [review]
LGTM! It's cool to initial this way to handle errors!

File: src/react/account/__tests__/Accounts.test.tsx (hunk 1 above)
Comment (JBWatenbergScality) [reply src/react/account/__tests__/Accounts.test.tsx]: done

**Pull request merged**